src/test/java/com/iota/iri/service/tipselection/impl/WalkValidatorImplTest.java

```diff
@@ -36,7 +36,7 @@
     @Mock
     private TransactionValidator transactionValidator;
     @Mock
-    private Milestone milestoneTracker;
+    private MilestoneTracker milestoneTrackerTracker;
```

Comment:
I believe this is a search/replace error. This private variable was already called milestoneTracker before the modification, and by replacing all milestone strings to milestoneTracker, now it is called milestoneTrackerTracker by mistake.